pkg/gadgettracermanager/k8s/k8s.go

```diff
@@ -72,3 +136,73 @@ func FillContainer(containerDefinition *pb.ContainerDefinition) error {
 
 	return nil
 }
+
+// PodToContainers return a list of the containers of a given Pod.
+// Containers that are not running or don't have an ID are not considered.
+func (k *K8sClient) PodToContainers(pod *v1.Pod) []pb.ContainerDefinition {
+	containers := []pb.ContainerDefinition{}
+
+	labels := []*pb.Label{}
+	for k, v := range pod.ObjectMeta.Labels {
+		labels = append(labels, &pb.Label{Key: k, Value: v})
+	}
+
+	for _, s := range pod.Status.ContainerStatuses {
```

Comment:
TODO (in a future PR): handle init containers. The info would be in `pod.Status.InitContainerStatuses` (similar struct as `pod.Status.ContainerStatuses`).